content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -0,0 +1,78 @@
+
+### Packer Integration and Artifact Registry
+
+Terraform Enterprise no longer includes an integrated Packer build system or an artifact registry for built images. Different organizations have such different requirements and expectations for image builds that we believe [running Packer out of your own CI/CD system](https://www.packer.io/guides/packer-on-cicd/index.html) will usually provide a better experience.
+
+### Periodically Queued Plans
+
+In legacy Terraform Enterprise, you could set an environment to periodically queue a plan regardless of whether the configuration had changed. This is gone, because [the run API](../api/run.html) makes it unnecessary.
+
+If you used scheduled runs because Terraform needed to run after events that Terraform Enterprise didn't know about, you can use your CI system to start runs in response to any kind of trigger. If you still want runs to happen on a fixed schedule, you can use any kind of scheduler to make an API call, including a cron job or a Nomad job.
+
+### Individual Collaborators on Environments
+
+Legacy Terraform Enterprise could grant access permissions to individual users; the new Terraform Enterprise handles all workspace permissions via [teams of users](../users-teams-organizations/teams.html).
+
+We removed individual access because it eventually makes your workspace settings a mess. Team-based access is easier to update and govern.
+
+### Personal Environment and Organization Variables
+
+Legacy Terraform Enterprise had a personal variables feature, where each user could set values that only applied to runs that they queued. In the new Terraform Enterprise, all variables are tied to a workspace, not a user.
+
+We removed this feature because it worked against Terraform Enterprise's goal of collaborative and predictable infrastructure. Not only did it make runs inconsistent and less reliable, it also expected that users would be queueing most runs manually in the UI, when we've found that people want most runs to happen in response to changes in a VCS repo or other automatic triggers.
+
+If you used personal variables to avoid sharing credentials, you can mark workspace variables as sensitive.
+
+### Terraform Push
+
+The new Terraform Enterprise does not support the `terraform push` command, which is now deprecated.
+
+This does not mean we're dropping support for a push-based workflow — we want push to be better, and the integrated `push` command was holding us back because we built it on an outdated understanding of how Terraform Enterprise should work.
+
+TFE's API supports uploading a configuration tarball and running it, and the [API-based run workflow page](../workspaces/run-api.html) explains how to integrate it into your automated systems. We also plan to release an external tool that mimics `terraform push`'s workflow soon.
```

Comment:
is the plan to update this section once the `terraform push` tool comes out this week? or are both coming out at the same time?

I would move this into the "Renamed or Moved Features" section since you get the same functionality, it's just called something new

Reply:
Yeah, I was gonna update it. Good point about moving it, although I might delay that until we've shipped the tool.

Reply:
I would move to "Moved" and also mention the deeper CLI integration plans.

Reply:
Agreed with armon.